### ISSUE TITLE
[FR] Add more anchor points to all nodes

### ISSUE THREAD

**Issue by Zottelchen** [Developer-Mike/obsidian-advanced-canvas#37]
Recently I've been trying to map out my social contacts and discovered that Obsidian only has 4 anchor points for each node. Turns out, this scenario is quite complex as most people have more than 4 connections. I would love to see more anchor points for all nodes.

**Comment by Developer-Mike** [comment]
There is already another FR mentioning this feature (#13). Is this similar to your request, or do you just want e.g. four anchor points per side?

**Comment by Zottelchen** [comment]
It could be a part of it 🤔 But for me (e.g.) 4 anchor points per side would be enough, I don't need to be able to have lines point "anywhere".

**Comment by Pixel3ro** [comment]
yeah. and they should auto align to the closest anchor. for example, you move your linked-card to the opposite side, the links should take the shortest distance.

**Comment by Developer-Mike** [comment]
This Feature would require changes to some core functions of canvas. Additionally, it would be very difficult to make it compatible with the open JSON Canvas format. Because of this, this won't get implemented.
